### ISSUE TITLE
False UnusedUse for generics

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#783]
```php
use My\Ns\Format

/**
 * @phpstan-template T of Format
 */
interface ResponseFormatter
```

`use` is reported by `SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse`

Current workaround seems to be 

```php
/**
 * @phpstan-template T of \My\Ns\Format
 */
interface ResponseFormatter

**Comment by kukulich** [comment]
Yeah, these annotations are ignored currently as discussed in https://github.com/slevomat/coding-standard/issues/736

**Comment by simPod** [comment]
@kukulich thanks for link
